process/project_milestones.md

```diff
@@ -57,6 +55,9 @@ Incubation milestones are guiding points to highlight success areas a maturing p
 * Project has engaged with a few Technical Advisory Groups to ensure they are considering best practices across multiple technical and governance facets
 * Project has defined a process for managing the conduct of its community, either establishing its own group to do this, leveraging the existing leadership, or deferring to the CNCF
 * Project maintains a dependency graph so adopters fully understand the complexity and risk of use
+* Project has 3 or more adopters using the project in production to support completion of Due Diligence and adopter interviews
+* Project plans activities to increase project adoption trajectory 3 adopters
+* Project has more than 5 regular contributors, more than half of which are from different organizations
```

Comment:
This one worries me A LOT. It clearly incentivizes projects and companies to do the wrong thing, in order to meet these "milestones".  

For example, fearing that they are being seen to have "too many" contributions going into the open source project, the major backer of a project decides to hold their engineering efforts back into closed-source add-ons. Or use outsourcing companies and partners rather than hire people to work full-time on the project. Or decide not to contribute a project to the CNCF because it's going to be such an uphill struggle for the project to see itself as successful according to these criteria. 

None of these outcomes are good for anyone in our community. 

(Also, why arbitrarily 5? And what does "regular" mean? We could spend time debating these points - but that is time that could be better spent on actually progressing / supporting projects.)

Reply:
These are good counterpoints @lizrice . We also don't want to have a single entity (company or organization or individual) be the majority decision maker of a projects technical direction, if that entity is acquired or disappears, the project can be left out to dry, and in some case unable to recover because the primary pool of contribution has evaporated.

@raravena80 @lizrice - How do we strike a balance in these milestones — as guiding points projects _could_ leverage to in order to become more resilient, mature, and widely adopted versions — that could help projects avoid the scenarios you present as well as a increase the likelihood of project's survivability with primary entity's complete absence?  What _actionable_ goals/ideals/structure/governance could be provided for projects to increase their resilience and survival likelihood in the event a core contributing entity ceases to be or continue? 

Ricardo mentioned open and flexible language here, would the following adjustments meet the intent?

```suggestion
* Project has multiple contributors who regularly participate in the growth and development of the project in accordance with the projects synchronous and asynchronous cadence
  * Project contribution governance defines timeliness and methods for recovering roles and permissions of contributors that drop from regular participation as _defined by the project_ (roles such as issue triage, community management, PR reviewer, maintainer, release lead, etc. and regularity such as participation in two of three releases, facilitation of 4 community meetups annually, etc.)
```

Reply:
I can't say I'm keen on removing specifics from this.  Folks in the community manager role for a project like having clear targets.  Once a project gets to applying Graduated, if they don't have at least five regular contributors, the application isn't likely to go well.  

Note that these are milestones, not updates to the specific requirements.

Reply:
👍  I believe all these can be covered in the [March 21st TOC meeting](https://docs.google.com/document/d/1jpoKT12jf2jTf-2EJSAl4iTdA7Aoj_uiI19qIaECNFc/edit#heading=h.8q8nrqluehva).